### PR TITLE
Darkstar mode

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: RiotSharp/Misc/GameMode.cs [hunk 1]
```diff
@@ -62,6 +62,11 @@ public enum GameMode
...
         /// </summary>
-        Assassinate
-    }
+        Assassinate,
+
+        /// <summary>
+        /// Darkstar game mode.
+        /// </summary>
+        Darkstar
+}
```
Comment (JanOuborny) [inline RiotSharp/Misc/GameMode.cs]: format

File: RiotSharp/MatchListEndPoint/Enums/Queue.cs [hunk 2]
```diff
@@ -56,8 +56,13 @@ public enum Queue
...
         /// </summary>
-        RankedPremade5x5
-    }
+        RankedPremade5x5,
+
+        /// <summary>
+        /// Darkstar games
+        /// </summary>
+        Darkstar
+}
```
Comment (JanOuborny) [inline RiotSharp/MatchListEndPoint/Enums/Queue.cs]: format

**Comment by BenFradet** [comment]
@JanOuborny fixed :+1:

**Comment by BenFradet** [comment]
I'll try to make the release tonight

**Pull request merged**